autobump-maven/src/test/java/com/github/autobump/maven/model/MavenDependencyBumperTest.java

```diff
@@ -0,0 +1,50 @@
+package com.github.autobump.maven.model;
+
+import com.github.autobump.core.model.Bump;
+import com.github.autobump.core.model.Dependency;
+import com.github.autobump.core.model.Version;
+import com.github.autobump.core.model.Workspace;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class MavenDependencyBumperTest {
+    private transient MavenDependencyBumper mavenDependencyBumper;
+    private transient Workspace workspace;
+    private transient MavenDependencyResolver resolver;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        mavenDependencyBumper = new MavenDependencyBumper();
+        Files.copy(Path.of("src/test/resources/project_root/testBump/pom.xml"), Path.of("src/test/resources/pom.xml"));
```

Comment:
Watch out with copying files in the source tree.
If possible, avoid this by using a temp directory or do these kind of operations in the build/ directory
But best not to do it in a directory under source control.